<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseWork</class>
 <widget class="QWidget" name="BaseWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QTableView" name="QuestView">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>331</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="AddC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>add</string>
   </property>
   <widget class="QLineEdit" name="lineEdit4AddID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4Add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit4Add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Content:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OKBut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancleBut">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="AddC_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>remove</string>
   </property>
   <widget class="QLineEdit" name="lineEdit4RemoveID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4Remove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit4Remove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Content:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton4Imp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>choose a file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
